tests/installation/welcome.pm

```diff
@@ -90,7 +90,7 @@ sub run {
 
     # license+lang +product (on sle15)
     # On sle 15 license is on different screen, here select the product
-    if (sle_version_at_least('15') && check_var('DISTRI', 'sle')) {
+    if (is_sle('15+') && check_var('DISTRI', 'sle')) {
```

Comment:
We don't need distribution check if using is_sle, which is whole point of it. `sle_version_at_least` returns true for non-sle distributions, so we had to check it explicitly. And thanks for taking care of existing code ;)